packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -2,13 +2,13 @@ openapi: 3.0.3
 info:
   title: PDND Interoperability API
   x-summary: PDND Interoperability API
-  version: "2.0.0"
+  version: '2.0.0'
```

Comment:
Could you revert formatting changes to this file? I think you may have a misconfigured formatter because my auto-format saves these files with double-quotes, which is the standard also for all other API spec files in the project